demo-ui/app/uk/gov/ons/addressIndex/demoui/views/index.scala.html

```diff
@@ -1,28 +1,27 @@
-@()(implicit messages: Messages)
 @import helper._
+@()(implicit messages: Messages)
 @*
  * Call the `main` template with two arguments. The first
  * argument is a `String` with the title of the page, the second
  * argument is an `Html` object containing the body of the page.
  *@
 
```

Comment:
These templates can be refactored heavily. Not for this PR. We should hold a server side templating session to improve this.